control-plane/tests/test_service_persistence.py

```
@@ -3898,6 +3928,97 @@ def test_service_analyst_queue_prefers_explicit_wazuh_source_for_multi_source_li
+            detail.lineage["substrate_detection_record_ids"],
+            ("wazuh:1731594986.4931506",),
+        )
+
+    def test_service_alert_detail_redacts_raw_native_payload_from_latest_reconciliation(
+        self,
+    ) -> None:
+        store, _ = make_store()
+        service = AegisOpsControlPlaneService(
+            RuntimeConfig(
+                postgres_dsn="postgresql://control-plane.local/aegisops",
+                wazuh_ingest_shared_secret="reviewed-shared-secret",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "wazuh_ingest_shared_secret"

(S106)